File: app/src/main/java/ng/com/techdepo/esidemtest/models/QuestionResponse.java
package ng.com.techdepo.esidemtest.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class QuestionResponse {

    @SerializedName("data")
    private List<Question> data;
    @SerializedName("status")
    private String status;
    @SerializedName("subject")
    private String subject;

    public List<Question> getData() {
        return data;
    }

    public void setData(List<Question> data) {
        this.data = data;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getSubject() {
        return subject;
    }

    public void setSubject(String subject) {
        this.subject = subject;
    }
}

File: app/src/main/java/ng/com/techdepo/esidemtest/models/QuestionResponse.kt
package ng.com.techdepo.esidemtest.models

import com.google.gson.annotations.SerializedName

class QuestionResponse {

    @SerializedName("data")
    var data: List<Question>? = null
    @SerializedName("status")
    var status: String? = null
    @SerializedName("subject")
    var subject: String? = null
}
